pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/DiscTorchToMhlo.cpp

```diff
@@ -0,0 +1,565 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
```

Comment:
Is rewrite patterns in this file newly created by us or copied from community repo?